release-notes-v3.0.md

```diff
@@ -0,0 +1,109 @@
+Intel® Neural Compressor v3.0 Release
+
+- Highlights
+- Features
+- Improvements
+- Examples
+- Bug Fixes
+- Documentations
+- External Contributions
+- Validated Configurations
+
+**Highlights**
+ - FP8 quantization and INT4 model loading support on [Intel® Gaudi® AI accelerator](https://habana.ai/products/gaudi2/) 
```

Comment:
How about FP8 quantization and weight-only INT4 quantization support on xxx?

Reply:
That's my previous draft, but I revised to this.  not sure if we have a clear alignment to claim INT4 WOQ support on xxx? since we only covers RTN/AutoRound without formal release test